packages/css-framework/src/components/_timeline.scss

```diff
@@ -4,7 +4,7 @@ $timeline-border-color: f.color("neutral", "100");
 $timeline-bg-color: f.color("neutral", "000");
 $timeline-alt-bg-color: f.color("neutral", "white");
 
-$day-width: 20px;
+$day-width: 30px;
```

Comment:
This should eventually come out of here and only happen in the component. It's confusing in here.